README.md

```diff
@@ -49,15 +41,27 @@ To build after all dependencies are installed:
     cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .
     make
 
-**NOTE:** BitShares requires an [OpenSSL](https://www.openssl.org/) version in the 1.0.x series. OpenSSL 1.1.0 and newer are NOT supported. If your system OpenSSL version is newer, then you will need to manually provide an older version of OpenSSL and specify it to CMake using `-DOPENSSL_INCLUDE_DIR`, `-DOPENSSL_SSL_LIBRARY`, and `-DOPENSSL_CRYPTO_LIBRARY`.
+**Upgrade Script** (prepend to the Build Script above if you built a prior release):
 
-**NOTE:** BitShares requires a [Boost](http://www.boost.org/) version in the range [1.57, 1.65]. Versions earlier than
-1.57 or newer than 1.65 are NOT supported. If your system Boost version is newer, then you will need to manually build
+    git remote set-url origin https://github.com/bitshares/bitshares-core.git
+    git checkout master
+    git remote set-head origin --auto
+    git pull
+    git submodule update --init --recursive # this command will fail
```

Comment:
`this command will fail` is not always true.

Reply:
change it to to "this command _may_ fail" will be best, else eliminate the comment?